truss/tests/test_model_inference.py

```diff
@@ -81,6 +82,58 @@ def test_binary_request(sklearn_rfc_model):
         assert np.shape(predictions["probabilities"]) == (1, 3)
 
 
+@pytest.mark.integration
+def test_model_load_failure_truss():
```

Comment:
do we want to create a variables of this test with `live_reload=True` to check the draft server behavior?

Reply:
>I am not including the integration test that models the live reload flow as the base image doesn't contain the control server updates which causes the test to fail. Running this locally to mimic the test produces the expected behavior. The remaining test can come via a follow up PR.

(from pr description)